charts/partials/templates/_network-validator.tpl

```diff
@@ -24,9 +24,13 @@ args:
   - --log-level
   - {{ .Values.networkValidator.logLevel }}
   - --connect-addr
-  - {{ .Values.networkValidator.connectAddr }}
+    {{- if .Values.disableIPv6 }}
+  - "1.1.1.1:{{ .Values.networkValidator.connectPort }}"
```

Comment:
So the only problem with this is that users can no longer modify the target now, right? This would probably change the behaviour of the validator. In air gapped environment, for example, if `1.1.1.1` is not accessible, the connection will fail. 

Can users still determine whether the failure was caused due to missing iptables rules as opposed to properties of their network? The validator once connected expects a random string to be returned from the server. Obviously with any other host that wouldn't happen, so if the connection succeeded but the actual payload was wrong, we deduce iptables is incorrectly set-up.

Do you think this is a problem? Chances are, the error will be different any way, but I worry that might remove from the strictness of the check.

Reply:
Those are great points. @mateiidavid and @lwj5. I'll add the additional config :+1: